doc/man3/flux_shell_plugstack_call.adoc

```diff
@@ -0,0 +1,39 @@
+flux_shell_plugstack_call(3)
+============================
+:doctype: manpage
+
+NAME
+----
+flux_shell_plugstack_call - Calls the function referenced by topic.
+
+
+SYNOPSIS
+--------
+ #include <flux/shell.h>
+
+ int flux_shell_plugstack_call (flux_shell_t *shell, 
+                                const char *topic, 
+                                flux_plugin_arg_t *args);
+
+DESCRIPTION
+-----------
+
+Wrapper for `plugstack_call()`.  Returns -1 and sets errno to EINVAL
```

Comment:
Expanding on the the shell's plugin implementation, and avoiding reference to an internal function:

The job shell implements a flexible plugin architecture which allows registration of one or more callback functions on arbitrary topic names. The stack of functions "listening" on a given topic string is called the "plugin stack". `flux_shell_plugstack_call()` exports the ability to call into the plugin stack so that plugins can invoke callbacks from other plugins.

Reply:
Done!